### DISCUSSION TITLE
Maybe not a bug, but it makes me uncomfortable

### DISCUSSION THREAD

**Discussion by MokOopsing** [osy86/HaC-Mini#959] [Q&A]
https://user-images.githubusercontent.com/39441028/141789386-c00fccdf-b453-4642-8d74-00e5e9cec885.mov

I don't know how to describe this "issue" (in English), is there anyone that got this  "issue" before, it appeared after I got a 4K display, it worked well on my old 2K display.

**Comment by Medizinmann-MD** [comment]
Well you mean like flickering/glitches of the screen?

This has been discussed – and there are several possible problems.

1st) Instability because of a bad cable – as simple as that…

2nd) Incompatibility of your Display with the way MacOS handles the signal.

3rd) Driver Problems of MacOS/HaC mini – here is the question – which version (of MacOS/HaCMini.pkg etc.) do you use?

Some of possible problems with 2. and 3. Haven been fixed in the past – i.e. see 
https://github.com/osy/HaC-Mini/issues/331#issue-703192210
https://github.com/osy/HaC-Mini/issues/526#issue-782933054

...some seem still to persist...
https://github.com/osy/HaC-Mini/issues/552#issue-837146992

**Comment by MokOopsing** [comment]
When I input password and login, the wallpaper became black, and there was a "loading", after seconds, everything went well

![image](https://user-images.githubusercontent.com/39441028/142412527-770dfac1-a0d4-4360-a6c2-844b3d161701.png)
